content/source/docs/cloud/integrations/kubernetes/index.html.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: "cloud"
+page_title: "Terraform Cloud Operator for Kubernetes setup instructions"
+description: |-
+  HashiCorp Terraform Cloud customers can integrate with Kubernetes using the official Terraform Cloud Operator for Kubernetes to provision infrastructure through Terraform Cloud using their Kubernetes control plane
+---
+
+# Terraform Cloud Operator for Kubernetes Setup Instructions
+
+-> **Note:** The Terraform Cloud Operator for Kubernetes is still under development and in the alpha testing stage. It is not ready for production use.
```

Comment:
```suggestion
~> **Important:** The Terraform Cloud Operator for Kubernetes is still under development and in the alpha testing stage. It is not ready for production use.
```

I reckon that's worth a yellow box instead of just a blue one.